build.gradle

```diff
@@ -23,18 +24,31 @@ tasks.withType(JavaCompile) {
 }
 
 dependencies {
-    compileOnly "org.embulk:embulk-core:0.9.23"
+    compileOnly "org.embulk:embulk-spi:0.10.31"
+    compileOnly "org.embulk:embulk-api:0.10.31"
+
+    compile("org.embulk:embulk-util-config:0.3.0") {
+        // They conflict with embulk-core. They are once excluded here,
+        // and added explicitly with versions exactly the same with embulk-core:0.10.19.
+        exclude group: "com.fasterxml.jackson.core", module: "jackson-annotations"
+        exclude group: "com.fasterxml.jackson.core", module: "jackson-core"
+        exclude group: "com.fasterxml.jackson.core", module: "jackson-databind"
+        exclude group: "com.fasterxml.jackson.datatype", module: "jackson-datatype-jdk8"
+        exclude group: "javax.validation", module: "validation-api"
+    }
+
     compile("io.github.medjed:JsonPathCompiler:0.0.12") {
         exclude group: "org.slf4j", module: "slf4j-api"
         exclude group: "org.apache.commons", module: "commons-lang3"
```

Comment:
The plugin should include `commons-lang3:3.4` as its dependencies. Look at other plugins using `commons-lang3`.

Reply:
Thanks. Fixed.